lib/audit.js

```diff
@@ -25,6 +31,63 @@ auditCmd.completion = function (opts, cb) {
   }
 }
 
+class Auditor extends Installer {
+  constructor (where, dryrun, args, opts) {
+    super(where, dryrun, args, opts)
+    this._deepArgs = (opts && opts.deepArgs) || []
+    this._runId = opts.runId || ''
+    this.audit = false
+  }
+
+  loadAllDepsIntoIdealTree (cb) {
+    BB.fromNode(cb => super.loadAllDepsIntoIdealTree(cb)).then(() => {
+      if (this._deepArgs && this._deepArgs.length) {
+        this._deepArgs.forEach(arg => {
+          arg.reduce((acc, child, i) => {
```

Comment:
Point of style, so please feel free to disregard: Personally I avoid single letter var names if they aren't used on the same line or next line, because they make search much harder. I would use `ii` here.

Reply:
noted :)